india_compliance/gst_india/overrides/subcontracting_transaction.py

```diff
@@ -314,8 +329,11 @@ def validate(self, doc, is_sales_transaction=False):
         self.validate_for_charge_type()
 
     def validate_for_same_party_gstin(self):
```

Comment:
after fixing `is_e_waybill_generatable_for_subcontracting`, this function may become redundant.